test/ci_workflow_gen.go

```diff
@@ -175,18 +169,12 @@ func clusterMySQLVersions(clusterName string) mysqlVersions {
 		return allMySQLVersions
 	case clusterName == "tabletmanager_tablegc":
 		return allMySQLVersions
-	case clusterName == "mysql80":
-		return []mysqlVersion{mysql80}
-	case clusterName == "vtorc_8.0":
-		return []mysqlVersion{mysql80}
-	case clusterName == "vreplication_across_db_versions":
-		return []mysqlVersion{mysql80}
+	case clusterName == "vtorc":
+		return allMySQLVersions
```

Comment:
This means that **two** workflow files are generated, one each for 5.7 and 8.0. This is why `vtorc_8.0` has been deleted from `clusterList`.